plugins/action/k8s_service.py

```diff
@@ -0,0 +1 @@
+k8s_info.py
```

Comment:
I didn't realize runtime.yml does not work on 2.9. @gravesm Thanks for catching this.